data/src/main/java/com/microsoft/azure/kusto/data/StringUtils.java

```diff
@@ -1,7 +1,11 @@
 package com.microsoft.azure.kusto.data;
 
-public class StringUtils {
-    public static String GetStringTail(String val, int minRuleLength) {
+public class StringUtils extends org.apache.commons.lang3.StringUtils {
```

Comment:
There are downsides to this approach, but we only want to add additional methods to the existing Apache Commons suite, so extending makes the most sense, especially to prevent name collisions/needing to add a package-qualified prefix to each call.